app/views/posts/show.html.erb

```diff
@@ -1 +1,29 @@
-<h1>Here is a list of posts for a given user</h2>
\ No newline at end of file
+<section class="show_user_posts">
+    <div class= "post-details post-border mt-2">
+            <h2>Post #<%= @user.id%> by: <%= @user.name%></h2>
+            <div class="comment-num">Comments: 10, Likes: 10k</div>
+        <p>
+            lorem ipsum dolor sit amet, consectetur adipisicing elit. lorem*ipsum dolor sit amet, consectetur adipisicing elit.
+            lorem ipsum dolor sit amet, consectetur adipisicing elit. lorem*ipsum dolor sit amet, consectetur adipisicing elit.
+            lorem ipsum dolor sit amet, consectetur adipisicing elit. lorem*ipsum dolor sit amet, consectetur adipisicing elit.
+            lorem ipsum dolor sit amet, consectetur adipisicing elit. lorem*ipsum dolor sit amet, consectetur adipisicing elit.
+            lorem ipsum dolor sit amet, consectetur adipisicing elit. lorem*ipsum dolor sit amet, consectetur adipisicing elit.
+            lorem ipsum dolor sit amet,\adipisicing elit. lorem*ipsum dolor sit amet, consectetur adipisicing elit.
+            lorem ipsum dolor sit amet, consectetur adipisicing elit. lorem*ipsum dolor sit amet, consectetur adipisicing elit.
+            lorem ipsum dolor sit amet, consectetur adipisicing elit. lorem*ipsum dolor sit amet, consectetur adipisicing elit.
+            dolor sit amet, consectetur adipisicing elit.
+            lorem ipsum dolor sit amet, consectetur adipisicing elit. lorem*ipsum dolor sit amet, consectetur adipisicing elit.
+
+        </p>
+    </div>
+    <div class="post-border mt-2">
+        <%(1..5).each do |i|%>
+            <div class="comment">
+                <h5>Anderson lebon<%= i %>: 
+                    <span> comment<%= i %></span>
+                </h5>
+            </div>
+        <% end %>
+    </div>
+</section>
```

Comment:
- [ ] I suggest that you don't add all of this in your show file remember this file is when you click on the user it should display a user page with 3recent posts of that specific user you can create add data in the seed file or add them manually using `rails console`
For example: `User.create(name: 'cool', photo: 'cool,png', bio: 'developer')`
after creating a user you can go ahead and create posts using that `user_id`

Reply:
- [ ] When you have users and posts you can be able to add comments and likes